ActionScript/ActionScript.sublime-syntax

```diff
@@ -2,9 +2,16 @@
 ---
 # http://www.sublimetext.com/docs/3/syntax.html
 name: ActionScript
+scope: source.actionscript.2
+
 file_extensions:
   - as
-scope: source.actionscript.2
+
+first_line_match: |-
+  (?xi:
+    ^ \s* // \s* -\*- .* \baction-?script\b .* -\*-  # editorconfig
```

Comment:
I think this is the only one with `.*` instead of `.*?`. Intentional or no?

Reply:
No.